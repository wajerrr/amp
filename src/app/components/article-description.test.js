import React from 'react';
import renderer from 'react-test-renderer';
import ArticleDescription from './article-description';

describe('ArticleDescription', () => {
  it('should match snapshot', () => {
    const tree = renderer
      .create(<ArticleDescription>description</ArticleDescription>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
