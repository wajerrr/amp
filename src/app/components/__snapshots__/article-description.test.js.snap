// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleDescription should match snapshot 1`] = `
<p
  className="css-1llzso0"
>
  description
</p>
`;
